#Thu Jul 21 09:22:54 CEST 2022
accesskey.nav=$org.olat.core.commons.fullWebApp\:accesskey.nav
accesskey.side=$org.olat.core.commons.fullWebApp\:accesskey.side
cancel=L\u00F6schen
error.addinfo=Zus\u00E4tzliche Informationen\:
error.back=Zur\u00FCck
error.businesspath=Business Pfad\:
error.businesspath.previous=Vorheriger Business Pfad\:
error.businesspath.unkown=Unbekannt
error.databaseexception=Unerwarteter Datenbankfehler.
error.errnum=Fehlernummer\:
error.header=Ups, da ging was schief\!
error.ioexception=Unerwarteter Ein-/Ausgabefehler.
error.login=Login
error.message1=Es ist ein Fehler aufgetreten, welcher vom System nicht automatisch behoben werden konnte. Wenn Sie den Support 
error.message2=kontaktieren m\u00F6chten, notieren Sie sich bitte folgende zus\u00E4tzliche Informationen.
error.report=Fehlerinformation
error.report.tell=Bitte teilen Sie uns mit, welche Aktion zu diesem Fehler gef\u00FChrt hat. Im untenstehenden Formular k\u00F6nnen Sie beschreiben, was Sie gemacht haben, bevor der Fehler aufgetreten ist oder welche Aktion zu dem Fehler gef\u00FChrt hat. Sie helfen uns damit, den Fehler zu finden und zu beheben, besten Dank\!<p /> Antwort erhalten Sie auf Ihre Mailadresse, die Sie in OpenOlat in Ihrem Profil angegeben haben.
error.securityexception=Unerwarteter Sicherheitsfehler\: Sie haben nicht gen\u00FCgend Berechtigungen f\u00FCr diese Aktion.
error.staleobjectexception=Das von Ihnen gew\u00E4hlte Objekt wurde in der Zwischenzeit von einer anderen Person gel\u00F6scht oder von Ihnen ver\u00E4ndert.
error.technical=Technische Informationen anzeigen
error.time=Datum und Zeit\:
send.report=Fehlerreport senden
